net/unbound/files/unbound.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+##############################################################################
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# Copyright (C) 2016 Eric Luehrsen
+#
+##############################################################################
+#
+# TODO: This file will build the UCI for Unbound. This iteration only puts
+# our default unbound configuration and root into /var/lib/unbound.
+#
+##############################################################################
+
+UNBOUND_B_MAN_CONF=1
+UNBOUND_B_DNSSEC=1
+
+UNBOUND_N_ROOT_AGE=30
+
+##############################################################################
+
+UNBOUND_VARDIR="/var/lib/unbound"
+UNBOUND_LIBDIR="/usr/lib/unbound"
+UNBOUND_PIDFILE="/var/run/unbound.pid"
+UNBOUND_CONFFILE="$UNBOUND_VARDIR/unbound.conf"
+UNBOUND_KEYFILE="$UNBOUND_VARDIR/root.key"
+UNBOUND_HINTFILE="$UNBOUND_VARDIR/root.hints"
```

Comment:
The quotes are not necessary the values given. On a full-blown system they'd be appropriate as paths can contain spaces, but this is OpenWrt/Lede and the paths are hardcoded.